File: app/src/main/java/br/com/popularmovies/services/movieService/response/MovieTrailers.java
package br.com.popularmovies.services.movieService.response;

import com.squareup.moshi.Json;

import java.util.List;

public class MovieTrailers {
    @Json(name = "results")
    private List<MovieTrailer> trailers;

    public List<MovieTrailer> getTrailers() {
        return trailers;
    }
}

File: app/src/main/java/br/com/popularmovies/services/movieService/response/MovieTrailers.kt
package br.com.popularmovies.services.movieService.response

import android.os.Parcelable
import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass
import kotlinx.android.parcel.Parcelize

@Parcelize
@JsonClass(generateAdapter = true)
data class MovieTrailers(@Json(name = "results") val trailerDtos: List<MovieTrailerDto>) : Parcelable